### PR TITLE
Removing color variables

### PR DESCRIPTION
**Pull request by stubelshe**
Needs approval from David.

### CODE REVIEW

File: src/styles/custom_variables.scss [hunk 1]
```diff
@@ -36,26 +36,4 @@ $additional-colors: (
   "grey-40": ($grey-40, $grey-40-inverse),
 );
 
-// navbar related variables
```
Comment (stubelshe) [inline src/styles/custom_variables.scss:39]: @datchley These were initial colors added for the boilerplate. We no longer use them, do you see any reason to keep them here?

Comment (ceane) [reply src/styles/custom_variables.scss:39]: @stubelshe I'd follow your instincts, if it doesn't look right, I'd go ahead and remove them.

Comment (stubelshe) [reply src/styles/custom_variables.scss:39]: Yeah, @datchley added these, just wanted his opinion here. My instinct is to yes, remove them.

**Comment by datchley** [comment]
@stubelshe be sure to rebase this branch against development to get the most recent changes on the `develop` branch before we merge this one.

**Pull request merged**